include/picongpu/particles/particleToGrid/derivedAttributes/EnergyDensityCutoff.def

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2013-2017 Axel Huebl, Rene Widera, Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/simulation_defines.hpp"
+#include "picongpu/traits/SIBaseUnits.hpp"
+#include <vector>
+
+#include "picongpu/particles/particleToGrid/derivedAttributes/EnergyDensity.def"
+
+
+namespace picongpu
+{
+namespace particleToGrid
+{
+namespace derivedAttributes
+{
+    /** Inherits from the `EnergyDensity` derived attribute
+     *
+     * @tparam T_ParamClass parameter class containing the maximum energy cutoff
+     *
+     * Note: `T_ParamClass` requires the member `float_X constexpr cutoffMaxEnergy`.
```

Comment:
`constexpr` to the left please :) (yes, it's an exception)

Reply:
whoops! right, didn't make that mistake in the code but in the comment ^^